riscv/src/runtime.rs

```diff
@@ -578,7 +615,7 @@ impl TryFrom<&[&str]> for Runtime {
                 continue;
             }
             match *name {
-                "poseidon_gl" => runtime = runtime.with_poseidon(),
+                "poseidon_gl" => runtime = runtime.with_poseidon_no_continuations(),
```

Comment:
Does this work because in the case of continuations we overwrite the syscall again?

Reply:
yea the idea is that by default we use this one, but in case of continuations we overwrite it

Reply:
actually it's wrong because internally it panics if a machine with this name is already there